pkg/server/environment_create.go

```diff
@@ -254,6 +256,13 @@ func getImageMeta(ctx context.Context, imageName string) (
 	if err != nil {
 		return
 	}
+	if size < 0 {
```

Comment:
I think the total size should be in the image metadata. Let's me check it.

Reply:
`image.Size()` always returns -1. Not sure why.